registry/purview-registry/registry/purview_registry.py

```diff
@@ -497,15 +497,14 @@ def _register_feathr_feature_types(self):
             force_update=True)
         logger.info("Feathr Feature Type System Initialized.")
 
-    def _upload_entity_batch(self, entity_batch):
+    def _upload_entity_batch(self, entity_batch:list[AtlasEntity]):
+        # we only support entity creation, update is not supported. 
+        # setting lastModifiedTS ==0 will ensure this, if another entity with ts>=1 exist
+        # upload funtion will fail with 412 Precondition fail.
```

Comment:
is there any code to handle 412 exception?

Reply:
This will be running in a standalone API project, on the api caller side, information will be 500. 
Will refine this to show 412 to the API caller, thanks for the advice! :P